region/maridia/outer.json

```diff
@@ -2315,9 +2489,28 @@
                       "shinesparkFrames": 150
                     }}
                   ]
+                },
+                {
+                  "name": "Cross Room Jump with Speedbooster, Spring Ball",
```

Comment:
Room names are put in notable strats to keep them unique.  It is probably not the best maintained system.